lib/zipflow/spec/store_entry.ex

```diff
@@ -26,8 +26,8 @@ defmodule Zipflow.Spec.StoreEntry do
     :zlib.close(ctx[:private][:z])
     frame = << 0x08074b50  :: size(32)-little,
                ctx[:crc]   :: size(32)-little,
-               ctx[:csize] :: size(32)-little,
-               ctx[:usize] :: size(32)-little
+               ctx[:csize] :: size(64)-little,
+               ctx[:usize] :: size(64)-little
```

Comment:
https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT

>      4.3.9.2 When compressing files, compressed and uncompressed sizes 
>      SHOULD be stored in ZIP64 format (as 8 byte values) when a 
>      file's size exceeds 0xFFFFFFFF.   However ZIP64 format MAY be 
>      used regardless of the size of a file.  When extracting, if 
>      the zip64 extended information extra field is present for 
>      the file the compressed and uncompressed sizes will be 8
>      byte values.  

When streaming we don't know upfront whether there will be any big files. Maybe the first file is < 4 Gb but the second can be > 4 Gb. However we have to choose if we're going to use 64-bit extensions or not when streaming the first file. So this code assumes to use ZIP64 in any case.